container-startup

```diff
@@ -5,15 +5,15 @@
 CMD=$1
 shift
 
-function usage() {
+usage() {
```

Comment:
can you elaborate why it is necessary ? ( generally I don't mind to remove it just want to know what is difference )

Reply:
Just to make the `shellcheck` tool happy:

```
In container-startup line 8:
function usage() {
^-- SC2112 (warning): 'function' keyword is non-standard. Delete it.
```

Reply:
ok , make sense :+1: